EventFilter/Phase2TrackerRawToDigi/test/BuildFile.xml

```diff
@@ -1,10 +1,24 @@
 <library file="plugins/*.cc" name="testEventFilterPhase2TrackerRawToDigiPlugins">
+  <use name="clhep"/>
```

Comment:
There are no changes in this package in this PR, so the changes in this `BuildFile.xml` should be removed (also from the commit history).